openfisca_uk/parameters/benefits/housing_benefit/means_test/earn_disregard.yaml

```diff
@@ -0,0 +1,8 @@
+description: Threshold in pounds per week from earnings per week, above which the contributory JSA amount is reduced equally.
```

Comment:
Should this read "housing benefit" rather than JSA? Similarly a few other descriptions seem to not match the folder structure.

Reply:
Yep, a lot of them are copied, I'll see which need to be corrected